planetary-social/nos
# Add name to profile card

## Issues covered
#1109

## Description
Adds the profile name to the card in Discover and Search.

## How to test
1. Navigate to Discover
2. Observe cards

## Screenshots/Video
![Simulator Screenshot - iPhone 15 Pro - 2024-05-15 at 17 13 36](https://github.com/planetary-social/nos/assets/59564/a64b92bb-299f-4d1c-91a1-12ad696f1938)
